README.textile

```diff
@@ -27,3 +27,39 @@ Get @digitalisiertedrucke@, set up the Play application, and run the tests:
 * @cd git/digitalisiertedrucke ; ~/activator-1.3.10-minimal/bin/activator test@
 
 See the @.travis.yml@ file for details on the CI config used by Travis.
+
+h1. Data import
+
+To transform the bzipped Marc-XML source data to JSON and index into an embedded Elasticsearch index, run:
+
+@~/activator-1.3.10-minimal/bin/activator "runMain ImportData"@
```

Comment:
How do I know when this is finished? Log is simply showing nothing new, right?
(Maybe add info to open a new shell or cancel running process for the next step )